### PR TITLE
refactor: remove license checking (call home)

### PR DESCRIPTION
**Pull request by socialmedialabs**
refactor: remove add-on update check (call home)
fix: add appropriate prefix according to connection (SSL or not)

### CODE REVIEW

**Comment by slang800** [comment]
This should be merged into master, not `delicious-brains`... `delicious-brains` is where the unaltered code from Delicious Brains is being kept

File: asset/dist/js/common-10161.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ function wpmdb_migration_type() {
 	var action = jQuery( 'input[name=action]:checked' );
 	if ( 0 === action.length ) {
 		return '';
-	}
```
Comment (socialmedialabs) [inline asset/dist/js/common-10161.js]: Somehow this indentation got lost. Do you think this will be a problem?

**Comment by socialmedialabs** [comment]
@slang800 may I ask if you actually plan on merging pull requests made to this branch? I understand that it can be quite an amount of work reviewing the code of a pull request, however, it would be a pitty if others (e. g. me) wouldn't be able to contribute to this fork at all.
Or is it supposed to work like that, have people fork your fork and others interested grab their copies from the forked forks?

**Comment by slang800** [comment]
Right now it has conflicts and a ton of other commits in it, so I haven't bothered reviewing yet.

**Comment by socialmedialabs** [comment]
Well, I basically from the plugin and its extensions took your delicous-brains branch and removed anything that would be related to licensing and home calling. 
I, however, didn't bother changing any variable or function names since I don't see any obligation to it, nor did I change the directory names. Hence I initially submitted my pull request to your delicious brains branch.
I have the complete bundle running in 4 staging and 1 production site without any problems.

**Pull request closed without merging**